lib/manager/helmv3/update.ts

```diff
@@ -20,7 +21,7 @@ export function bumpPackageVersion(
     }
     logger.debug({ newChartVersion });
     bumpedContent = content.replace(
-      /^(?<version>version:\s*).*$/m, // TODO #12070
+      regEx(`^(?<version>version:\\s*).*$`, 'm'), // TODO #12071
```

Comment:
This will replace any comments after the version, we should open a new issue

Reply:
Removing `.*` from the regex solves the issue.

Updated: ```   regEx(`^(?<version>version:\\s*)$`, 'm'), ```